test/integration-tests/UniversalRouter.test.ts

```diff
@@ -149,225 +117,37 @@ describe('UniversalRouter', () => {
-        ).connect(alice) as UniversalRouter
-
-        miladyContract = MILADY_721.connect(alice) as ERC721
-
-        // add valid nftx order to planner
-        nftxValue = expandTo18DecimalsBN(2.036523961400441269)
-        numMiladys = 1
-        const calldata = nftxZapInterface.encodeFunctionData('buyAndRedeem', [
-          NFTX_MILADY_VAULT_ID,
-          numMiladys,
-          [],
-          '0xd9627aa400000000000000000000000000000000000000000000000000000000000000800000000000000000000000000000000000000000000000001bfb8d0ff32c43470000000000000000000000000000000000000000000000000e27c49886e6000000000000000000000000000000000000000000000000000000000000000000010000000000000000000000000000000000000000000000000000000000000002000000000000000000000000c02aaa39b223fe8d0a0e5c4f27ead9083c756cc2000000000000000000000000227c7df69d3ed1ae7574a1a7685fded90292eb48869584cd00000000000000000000000010000000000000000000000000000000000000110000000000000000000000000000000000000000000000465b3a7f1b643618cb',
-          alice.address,
-        ])
-        planner.addCommand(CommandType.NFTX, [nftxValue, calldata])
-
-        let invalidSeaportOrder = JSON.parse(JSON.stringify(seaportV1_4Orders[1]))
-        invalidSeaportOrder.protocol_data.signature = '0xdeadbeef'
-        let seaportOrder: Order
-        ;({ advancedOrder: seaportOrder, value: seaportValue } = getAdvancedOrderParams(invalidSeaportOrder))
-        invalidSeaportCalldata = seaportInterface.encodeFunctionData('fulfillAdvancedOrder', [
-          seaportOrder,
-          [],
-          OPENSEA_CONDUIT_KEY,
-          alice.address,
-        ])
-
-        value = seaportValue.add(nftxValue)
-      })
-
-      it('reverts if no commands are allowed to revert', async () => {
-        planner.addCommand(CommandType.SEAPORT_V1_4, [seaportValue, invalidSeaportCalldata])
-
-        const { commands, inputs } = planner
-
-        const testCustomErrors = await (await ethers.getContractFactory('TestCustomErrors')).deploy()
-        const customErrorSelector = findCustomErrorSelector(testCustomErrors.interface, 'InvalidSignature')
-        await expect(router['execute(bytes,bytes[],uint256)'](commands, inputs, DEADLINE, { value }))
-          .to.be.revertedWithCustomError(router, 'ExecutionFailed')
-          .withArgs(1, customErrorSelector)
-      })
-
-      it('does not revert if invalid seaport transaction allowed to fail', async () => {
-        planner.addCommand(CommandType.SEAPORT_V1_4, [seaportValue, invalidSeaportCalldata], true)
-        const { commands, inputs } = planner
-
-        const miladyBalanceBefore = await miladyContract.balanceOf(alice.address)
-        await router['execute(bytes,bytes[],uint256)'](commands, inputs, DEADLINE, { value })
-        const miladyBalanceAfter = await miladyContract.balanceOf(alice.address)
-        expect(miladyBalanceAfter.sub(miladyBalanceBefore)).to.eq(numMiladys)
-      })
-    })
+    it('reverts if no commands are allowed to revert')
```

Comment:
unrun test cases?

Reply:
yeeah need to completely rewrite them as they were all written for NFTs